install/kubernetes/cilium/values.yaml

```diff
@@ -437,12 +437,15 @@ hubble:
   # Enable Hubble (true by default).
   enabled: true
 
-  # Buffer size of the channel Hubble uses to receive monitor events. If this value is not set,
-  # the queue size is set to the default monitor queue size.
+  # Buffer size of the channel Hubble uses to receive monitor events. If this
+  # value is not set, the queue size is set to the default monitor queue size.
   # eventQueueSize: ""
```

Comment:
nit: `eventQueueSize` is an integer, not a string, and so the value should be `0` or empty, not the empty string `""`.

Reply:
This is unrelated to changes in this PR. For the string though, same comment as for `eventBufferCapacity` applies.